packages/calcite-components/src/demos/input-time-zone.html

```diff
@@ -34,23 +34,470 @@
     </style>
     <script src="./_assets/head.js"></script>
     <script type="module" src="./_assets/demo-spacer.js"></script>
+    <script>
+      (function () {
+        const originalSupportedValuesOf = Intl.supportedValuesOf;
+
+        Intl.supportedValuesOf = function (key) {
+          if (key === "timeZone") {
+            /**
+             * Polyfilled due to lack of support for `Intl.DateTimeFormatOptions#timeZoneName` in Chromium v92 (bundled in Puppeteer v10).
+             */
+            const fallbackTimeZoneNames = [
```

Comment:
nitpick: Could this be saved in some other JS file? could maybe be reusable.

Reply:
Whoopsie! This was meant to the time zone polyfill used in E2E tests. Removing! 

![](https://media.tenor.com/zu4gNl1O9g0AAAAC/penguin-hide.gif)

> could maybe be reusable.

Possibly, but it may also not be needed once we bump Stencil in November. I think E2E is the only place where this is needed.